src/Sylius/Bundle/ApiBundle/Command/Payment/UpdatePaymentRequest.php

```diff
@@ -13,31 +13,15 @@
 
 namespace Sylius\Bundle\ApiBundle\Command\Payment;
 
+use Sylius\Bundle\ApiBundle\Attribute\PaymentRequestHashAware;
 use Sylius\Bundle\ApiBundle\Command\IriToIdentifierConversionAwareInterface;
-use Sylius\Bundle\ApiBundle\Command\PaymentRequestHashAwareInterface;
 
-/** @experimental */
-class UpdatePaymentRequest implements PaymentRequestHashAwareInterface, IriToIdentifierConversionAwareInterface
+#[PaymentRequestHashAware]
+class UpdatePaymentRequest implements IriToIdentifierConversionAwareInterface
 {
-    protected ?string $hash = null;
-
     public function __construct(
-        private mixed $payload = null,
+        public ?string $hash = null,
```

Comment:
Shouldn't this field be obligatory and immutable?
```suggestion
        public readonly string $hash,
```